app/views/activity_forms/purpose.html.haml

```diff
@@ -1,10 +1,11 @@
 = render layout: "wrapper" do |f|
+  - activity_presenter = ActivityPresenter.new(@activity)
```

Comment:
I don't usually like creating variables like this in the view, but I think in this case it's fine and keeps the controller tidier 🤷

Reply:
It's a little different to how we do it with `:implementing_organisation`, which sets an instance variable in both `show` and `update`, but this way is DRYer

Has it been done in other activity form views? If so, I think it's probably okay as long as we don't set a bunch of variables directly in views